src/Cooked/MockChain/GenerateTx/Body.hs

```diff
@@ -59,6 +60,11 @@ instance Transform TxContext Input.InputContext where
 instance Transform TxContext Collateral.CollateralContext where
   transform TxContext {..} = Collateral.CollateralContext {..}
 
+instance Transform TxContext Withdrawals.WithdrawalsContext where
```

Comment:
I am a bit skeptical about this `Transform` typeclass. This is a whole topic on its own, not related to this here specifically, so I won't comment on this here anymore. We may discuss it between maintainers or open the discussion in an issue.